data/Language/Flow.yaml

```diff
@@ -0,0 +1,51 @@
+identifier: Flow
+full_name: Flow
+wikidata: Q59413589
```

Comment:
https://www.wikidata.org/wiki/Q59413589 404's.

Reply:
The page existed previously.
However, it was removed recently.
Updated. Thank you!